test/unit/services/test_catalogue_category.py

```diff
@@ -307,6 +356,78 @@ def test_create_with_duplicate_property_names(catalogue_category_repository_mock
     )
 
 
+def test_create_catalogue_item_properties_with_non_existent_unit_id(
```

Comment:
Sorry just spotted, we do, do invalid ids for service tests in some cases, it would be like test_create_with_invalid_catalogue_category_id in items, so they can be done here.